edi_record_metadata_oca/tests/test_metadata.py

```diff
@@ -27,6 +27,7 @@ def _setup_records(cls):
             direction="output",
         )
         cls.exc_record = cls.backend.create_record(cls.exc_type.code, {})
+        return res
```

Comment:
I have changed it because of the `pre-commit`
![image](https://github.com/QuocDuong1306/edi-framework/assets/127363167/63527c36-cf07-45a4-adef-c0f12dd75e58)